keyboards/torn/torn_encoder.c

```diff
@@ -52,7 +50,9 @@ bool encoder_update_kb(uint8_t index, bool clockwise) {
       code = encoder_default[index][clockwise];
     }
 
-    tap_code16(code);
+    if (encoder_update_user(code, clockwise) == false) {
```

Comment:
Logic is inverted.  False is that the kb code should NOT be respected. 
```suggestion
    if (encoder_update_user(code, clockwise)) {
```